src/sidebar/store/modules/groups.ts

```diff
@@ -7,12 +7,10 @@ import type { State as SessionState } from './session';
 
 type GroupID = Group['id'];
 
-/**
- * `null`: Members not yet loaded for currently focused group
- * 'loading': Members being currently loaded
- * GroupMember[]: Members already loaded
- */
-export type FocusedGroupMembers = null | 'loading' | GroupMember[];
+export type FocusedGroupMembers =
+  | { status: 'not-loaded' }
+  | { status: 'loading' }
+  | { status: 'loaded'; members: GroupMember[] };
```

Comment:
There is no error state here. What happens currently if group members fail to load for any reason?

Reply:
I reckon it would end up in progress indefinitely. I'll address that in a follow-up PR